views/includes/check-your-answers/legal-entity-trustee.html

```diff
@@ -94,9 +94,9 @@
     },
     actions: {
     items: [ CREATE_CHANGE_LINK(
-        OE_CONFIGS.TRUSTS_URL +  "/" + trust.trust_id + "/" + OE_CONFIGS.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_PAGE + "/" + legalEntity.id + OE.CONFIGS.INTERESTED_PERSON_START_DATE,
-        "Individual beneficial owner " + legalEntity.name + " - date of birth",
-        "change-individual-beneficial-owner-interested-person-date-button"
+        OE_CONFIGS.TRUSTS_URL +  "/" + trust.trust_id + "/" + OE_CONFIGS.TRUST_LEGAL_ENTITY_BENEFICIAL_OWNER_PAGE + "/" + legalEntity.id + OE.CONFIGS.INTERESTED_PERSON_START_DATE,
+        "Individual beneficial owner " + legalEntity.name + " -date became an interested person",
```

Comment:
Did you mean to delete the space before date?

Reply:
edited